crates/fuel-core/src/query/coin.rs

```diff
@@ -63,4 +72,16 @@ impl ReadView {
             })
             .try_flatten()
     }
+
+    pub fn coins_to_spend(
+        &self,
+        owner: &Address,
+        asset_id: &AssetId,
+        max: u16,
+    ) -> Result<Vec<CoinType>, CoinsQueryError> {
+        let _coin_ids = self.off_chain.coins_to_spend(owner, asset_id, max);
```

Comment:
Do we need to call it, if below we have todo?

Reply:
This piece of code is now removed.